src/microbe_stage/MicrobeInternalCalculations.cs

```diff
@@ -191,8 +199,14 @@ public static float CalculateCapacity(IEnumerable<OrganelleTemplate> organelles)
                 rightwardDirectionMovementForce += MovementForce(movementConstant, rightDirectionFactor);
                 leftwardDirectionMovementForce += MovementForce(movementConstant, leftDirectionFactor);
             }
+
+            massEstimate += averageDensity * organelle.Definition.HexCount;
         }
 
+        // If this estimate could be made more accurate without additional computation that would be great
```

Comment:
Why doesn't the estimate use just the density of each organelle as it is processed above? Organelle mass estimates could be precalculated inside the OrganelleDefinition load to reduce runtime calculations.